_topic_map.yml

```diff
@@ -111,6 +111,8 @@ Topics:
 - Name: Managing user access
   Dir: manage-user-access
   Topics:
+  - Name: Configuring G Suite as an OIDC identity provider
```

Comment:
Just checking if it has to be G Suite, or if it could be another OIDC IDP?

Reply:
I've changed it to Google Workspace (Thank you). Its only for this particular provider.